src/librustc/infer/error_reporting/mod.rs

```diff
@@ -963,4 +963,5 @@ impl<'tcx> ObligationCause<'tcx> {
             _ => "types are compatible",
         }
     }
-}
+
+}
```

Comment:
Missing new line at the end of the file and you added an empty line before this one.